### DISCUSSION TITLE
About Icons

### DISCUSSION THREAD

**Discussion by KuroProjects** [spencerwooo/onedrive-vercel-index#534] [Q&A] [answered]
Icons in the index is not working

also does that feature will work where instead of putting the image in onedrive 
we use image from web and put image address in the required field

**Comment by zkeq** [comment, marked as answer]
Can you provide your config file?
It looks like my working icons are like this:

```url
https://tuostudy.com/_next/image/?url=%2Ffavicon.ico&w=32&q=75
```

And your site seems like this.

```url
https://cherryxhenil.vercel.app/_next/image/?url=Cherry%2Ficons%2F128.png&w=32&q=75
```

Seems like `%2F` is affecting it?

```
?url=%2F
```

Looks like this file is working. Maybe you should check the config.

```
https://cherryxhenil.vercel.app/_next/image/?url=%2Ficons%2F128.png&w=32&q=75
```

**Reply by KuroProjects** [reply]
i put it like that


  icon: "Cherry/icons/128.png",

**Reply by KuroProjects** [reply]
Well sorry it was my mistake
i actually didn't put that in repository
instead put it in onedrive

**Reply by zkeq** [reply]
> i put it like that
> 
> icon: "Cherry/icons/128.png",

Where do you put your `icon` file? It should be placed in the `public` directory ..

**Reply by KuroProjects** [reply]
thanks for the help though

**Reply by KuroProjects** [reply]
> 

btw how did you got icons even after your files name in the index?
i am kind of confused in that
